docs/sources/release-notes/v2-9.md

```diff
@@ -34,29 +34,38 @@ Grafana Labs is excited to announce the release of Loki 2.9.0 Here's a summary o
  
 ## Bug fixes
 
+### 2.9.5 (2024-02-28)

```

Comment:
Should [2.9.6](https://github.com/grafana/loki/releases/tag/v2.9.6) be here as well?

Reply:
It's listed as DRAFT?